hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/SplitLogWorker.java

```diff
@@ -104,7 +104,8 @@ static Status splitLog(String filename, CancelableProgressable p, Configuration
     // encountered a bad non-retry-able persistent error.
     try {
       SplitLogWorkerCoordination splitLogWorkerCoordination =
-        server.getCoordinatedStateManager().getSplitLogWorkerCoordination();
```

Comment:
Can splitLogWorkerCoordination be null and the reason? Thanks.

Reply:
splitlogworkercoordination is only present when we are doing the old zk based log splitting. Otherwise, it is null.

Reply:
+1 on this fix, I hit this issue and figure out if we set `HBASE_SPLIT_WAL_COORDINATED_BY_ZK` in `TestClusterRestartFailoverSplitWithoutZk`, the test fails.